README.md

```diff
@@ -94,7 +94,7 @@ Requirements:
 Run:
 
 ```
-go get github.com/taskcluster/slug-go/slug
+go get github.com/taskcluster/slugid-go
```

Comment:
Ah yes, it was wrong. But I think it should be

```go
go get github.com/taskcluster/slugid-go/slug
```

The top level folder doesn't contain go files directly, only the subfolders, so I believe we need the extra `/slug`.